README.md

```diff
@@ -1,320 +1,41 @@
-  inf-triton    Perform inference with Triton
-  mlflow-drift  Report model drift statistics to ML Flow
-  monitor       Display throughput numbers at a specific point in the pipeline
-  preprocess    Convert messages to tokens
-  serialize     Serializes messages into a text format
-  to-file       Write all messages to a file
-  to-kafka      Write all messages to a Kafka cluster
-  validate      Validates pipeline output against an expected output
-```
-
-And for the FIL pipeline:
-
-```bash
-$ morpheus run pipeline-fil --help
-Usage: morpheus run pipeline-fil [OPTIONS] COMMAND1 [ARGS]... [COMMAND2
-                               [ARGS]...]...
+# NVIDIA Morpheus
 
-<Help Paragraph Omitted>
+NVIDIA Morpheus is an open AI application framework that provides cybersecurity developers with a highly optimized AI framework and pre-trained AI capabilities that allow them to instantaneously inspect all IP traffic across their data center fabric. The Morpheus developer framework allows teams to build their own optimized pipelines that address cybersecurity and information security use cases. Bringing a new level of security to data centers, Morpheus provides development capabilities around dynamic protection, real-time telemetry, adaptive policies, and cyber defenses for detecting and remediating cybersecurity threats.
 
-Commands:
-  add-class     Add detected classifications to each message
-  add-scores    Add probability scores to each message
-  buffer        (Deprecated) Buffer results
-  delay         (Deprecated) Delay results for a certain duration
-  deserialize   Deserialize source data from JSON.
-  dropna        Drop null data entries from a DataFrame
-  filter        Filter message by a classification threshold
-  from-file     Load messages from a file
-  from-kafka    Load messages from a Kafka cluster
-  inf-identity  Perform a no-op inference for testing
-  inf-pytorch   Perform inference with PyTorch
-  inf-triton    Perform inference with Triton
-  mlflow-drift  Report model drift statistics to ML Flow
-  monitor       Display throughput numbers at a specific point in the pipeline
-  preprocess    Convert messages to tokens
-  serialize     Serializes messages into a text format
-  to-file       Write all messages to a file
-  to-kafka      Write all messages to a Kafka cluster
-  validate      Validates pipeline output against an expected output
-```
+## Documentation
+### Using Morpheus
+* [Getting Started with Morpheus](./docs/source/getting_started.md) - Using pre-built Docker containers, building Docker containers from source, and fetching models and datasets
+* [Morpheus CLI Overview](./docs/source/basics/overview.rst) - Brief overview of the `morpheus` command line interface
+* [Building a Pipeline](./docs/source/basics/building_a_pipeline.rst) - Introduction to building a pipeline using the command line interface
+* [Morpheus Examples](./examples.md) - Example pipelines using both the Python API and command line interface
```

Comment:
Link to Morpheus Examples readme is broken
suggest change (./examples.md) to (./examples)